source/transfer_list.rst

```diff
@@ -98,6 +100,44 @@ changes will be backwards-compatible to older readers.
      - 0xc
      - The maximum number of bytes that the TL can occupy. Any entry producer must check if there is sufficient space before adding an entry to the list. Firmware can resize and/or relocate the TL and update this field accordingly, provided that the TL requirements are respected. This field must be a multiple of 8.
 
+   * - flags
+     - 0x4
+     - 0x10
+     - Flags word. See below for contents.
+
+   * - reserved
+     - 0x4
+     - 0x14
+     - Reserved word. Must be set to 0 or ignored.
+
+
+TL Flags
+^^^^^^^^
+
+The TL flags word is intended to signal properties relating to the TL as a
+whole. Future flag values may be added according to the rules of the `version`
+field.
+
+The TL flags word is intended to signal properties relating to the TL as a
+whole. Future flag values may be added according to the rules of the `version`
```

Comment:
This text seems to appear twice?

Reply:
The text duplication still apears in the latest commit. I'll "unresolve" the comment so we don't lose track.

Reply:
It might be a rebase issue. I'm not sure, but it seems gone for me